src/Configurator/ContainerConfigurator.php

```diff
@@ -76,13 +76,31 @@ private function addParameters(array $parameters)
             $parametersLines[] = "parameters:\n";
         }
         foreach ($parameters as $key => $value) {
-            // FIXME: var_export() only works for basics types, but we don't have access to the Symfony YAML component here
-            $parametersLines[] = sprintf('    %s: %s%s', $key, var_export($value, true), "\n");
+            if (\is_array($value)) {
+                $parametersLines[] = $this->parse(1, $value);
+                continue;
+            }
+            $parametersLines[] = sprintf("    %s: '%s'%s", $key, $value, "\n");
         }
         if (!$endAt) {
             $parametersLines[] = "\n";
         }
         array_splice($lines, $endAt, 0, $parametersLines);
         file_put_contents($target, implode('', $lines));
     }
+
+    private function parse($level, $array): string
```

Comment:
`dumpYaml()`? could deserve some test cases?

Reply:
There was still some bug that are now corrected and covered by tests 😃